src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
+
+|Keys|Action|
+|---|---|
+|`Ctrl` + `K` or `⌘` + `K`| access command center|
+|`ENTER`|execute command center command|
+
+# Shortcuts
+
+Only in the Appwrite SDK console use these keyboard shortcuts to access Appwrite's SDK console.  Using the first letter from the call to action and the resource/product/service/page, Appwrite's keyboard shortcuts follow this pattern.  For example, the shortcut `G` then `S` goes to storage, but if you want to go to settings, the shortcut is `G` then `E` because `G` then `S` is already taken, so the next letter of settings is used.
+
+|Shortcut|Action|
```

Comment:
We discussed to have this as two sections unless we can think of more, Global shortcuts (user account for example) and then project level ones.